app/Http/Middleware/CASAuthenticate.php

```diff
@@ -74,13 +72,8 @@ public function handle(Request $request, Closure $next)
                     $this->cas->setAttributes($masq_attrs);
                 }
 
-                if (config('features.demo-mode') === null) {
-                    AuthStickler::check($this->cas);
-                } else {
-                    if ($this->cas->user() !== config('features.demo-mode')) {
-                        Unauthorized::render(0);
-                        exit;
-                    }
+                if (config('features.demo-mode') !== null && $this->cas->user() !== config('features.demo-mode')) {
```

Comment:
Is `features.demo-mode` normally set to a username?

Reply:
Only in the Google Play review environment; it is null in test and prod.